runtime/jcl/common/sun_misc_Perf.c

```diff
@@ -39,7 +40,7 @@ Java_sun_misc_Perf_registerNatives(JNIEnv *env, jclass klass)
 jobject JNICALL
 Java_sun_misc_Perf_attach(JNIEnv *env, jobject perf, jstring user, jint lvmid, jint mode)
 {
-	return NULL;
+	assert(!"Java_sun_misc_Perf_attach unimplemented");
```

Comment:
The return statement must stay, else we should expect compile errors:
```
	assert(!"Java_sun_misc_Perf_attach unimplemented");
	return NULL;
```
The same goes for `Java_jdk_internal_perf_Perf_attach0`.